tests/rest/RestBase.php

```diff
@@ -43,7 +42,7 @@
  * @requires extension curl
  * @group REST
  */
-abstract class RestBase extends TestCase {
+abstract class RestBase extends PHPUnit\Framework\TestCase {
```

Comment:
Why are you removing the import in favor of a qualified reference ? I'd rather not change that.

Reply:
Well, I just gave it the look and feel used in other MantisBT tests, plus the imported name is only used once and it is very close to the beginning of the file and immediately visible to the developer.

Reply:
Sorry for picking nits, but I insist... you're just reverting a change that I made in 2023. 

https://github.com/mantisbt/mantisbt/commit/829ab70c25cbfac36efccb4a80eac9bb46f38ac5#diff-b52215be84fb8db024faf46fcfa124b282a4b141f1327588fc78660f3c36d980L43-R46

> I just gave it the look and feel used in other MantisBT tests

In that case it's the *other* test scripts that should abandon the fully qualified refs; I usually fix those things when I need to update a file, which explains the discrepancy - but there should not be that many of them.

Reply:
Thanks

> I have seen in many places in MantisBT the use of fully qualified names, e.g. function parameter types...

I'm slowly getting rid of them, whenever I notice an occurrence.